### DISCUSSION TITLE
RUN_ON_STARTUP environment variable, what does it do?

### DISCUSSION THREAD

**Discussion by notflip** [borgmatic-collective/docker-borgmatic#289] [Q&A] [answered]
I'm wondering what the `RUN_ON_STARTUP` flag does? How does it influence the docker container?

**Comment by witten** [comment, marked as answer]
It looks like it does just what it says on the tin: https://github.com/borgmatic-collective/docker-borgmatic/blob/2a56a5b9ba0ee6a9c7806d7123a8d9cd2c63de55/entry.sh#L79

It runs borgmatic when the container starts up instead of (just) waiting for a cron job to kick in.

**Reply by notflip** [reply]
Ah right, forgot to check there. Thanks!
